src/lib/transition/slide.ts

```diff
@@ -5,41 +5,39 @@ import type { EasingFunction } from 'svelte/transition';
  * Slide transition that supports multiple directions.
```

Comment:
Interesting, apparently here svelte uses snake_case var names (since if I remember correctly I just copied it and added the direction param because I needed it for the animation you can see in the docs sidebar). I don't mind this change though.